anonympy/pdf/utils_pdf.py

```diff
@@ -1,113 +1,116 @@
 import os
 import re
+from typing import Dict, List, Tuple
+
 import cv2
-from typing import List, Tuple, Dict
-from PIL import ImageDraw, Image
+from PIL import Image, ImageDraw
 from PyPDF2 import PdfFileMerger
 
 
 def find_EOI(pipeline, matches: list, EOI: str) -> None:
     for obj in pipeline:
-        group = obj['entity_group']
-        word = obj['word']
-        if group == EOI and (len(word.strip('#')) > 3):
-            temp = word.strip('#').split(' ')
-
-            for w in temp:
-                if len(w) > 2:
-                    matches.append(w)
+        group = obj["entity_group"]
+        word = obj["word"]
+        if group == EOI and len(word.strip("#")) > 3:
+            temp = word.strip("#").split()
+            [matches.append(w) for w in temp if len(w) > 2]
 
 
-def draw_boxes_easyOCR(image: Image.Image,
-                       bounds: list,
-                       color: str = 'yellow',
-                       width: int = 2):
+def draw_boxes_easyOCR(
+    image: Image.Image, bounds: list, color: str = "yellow", width: int = 2
+):
```

Comment:
Really? Isn't the first version visually more pleasing?

Reply:
I don't know) I just let linters do that for me. I suppose, single line is not too long to separate all args